rocAL/include/decoders/image/decoder.h

```diff
@@ -34,11 +34,6 @@ THE SOFTWARE.
 #include "hip/hip_runtime.h"
 #endif
 
-#if ENABLE_HIP
-#include "hip/hip_runtime_api.h"
-#include "hip/hip_runtime.h"
-#endif
-
 enum class DecoderType {
```

Comment:
I think, we can keep all enum names without _DECODE at the end for consistency

Reply:
You can make it as a new PR. I suggest changing the names to FFMPEG_HOST, ROCDEC_VIDEO_HW, AUDIO_SNDFILE, ROCJPEG_HW, ROCJPEG_CORPPED_HW